ext/natives/codegen_out_cs.lua

```diff
@@ -4,6 +4,25 @@ local function printFunctionName(native)
 	end)
 end
 
+function table.shallow_copy(t)
```

Comment:
I see that this and other Nits were closed without any changes to the code. Is there any reasons you disagree with them? They are minor, but I think it would be good to fix before merging.